python/lsst/daf/butler/formatters/parquet.py

```diff
@@ -231,21 +231,28 @@ def arrow_to_numpy(arrow_table: pa.Table) -> np.ndarray:
     -------
     array : `numpy.ndarray` (N,)
         Numpy array table with N rows and the same column names
-        as the input arrow table.
+        as the input arrow table. Will be masked records if any values
+        in the table are null.
```

Comment:
I guess the return type should be `np.ndarray` or `np.ma.MaskedArray`?  Same for the mypy type?